core/types/block.internal.libevm_test.go

```diff
@@ -0,0 +1,42 @@
+package types
+
+import (
+	"reflect"
+	"slices"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_Body_hasOptionalFields(t *testing.T) {
+	t.Parallel()
+
+	var body Body
+
+	assert.False(t, body.hasOptionalFields(), "body has no optional field set")
+
+	v := reflect.ValueOf(&body).Elem()
+	typ := reflect.TypeOf(body)
+	for i := 0; i < v.NumField(); i++ {
+		fieldType := typ.Field(i)
+		rlpTag := fieldType.Tag.Get("rlp")
+		rlpTagFields := strings.Split(rlpTag, ",")
+		if !slices.Contains(rlpTagFields, "optional") {
```

Comment:
```suggestion
		tag := fieldType.Tag.Get("rlp")
		if !slices.Contains(strings.Split(tag, ","), "optional") {
```

We know from the RHS that it's an RLP tag so that part of the variable name is redundant (since there's no usage beyond the next line). The intermediate variable IMO just wastes brain-stack space.